hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/client/ScmClient.java

```diff
@@ -331,7 +332,8 @@ Pipeline createReplicationPipeline(HddsProtos.ReplicationType type,
   /**
    * Start ContainerBalancer.
    */
-  boolean startContainerBalancer(Optional<Double> threshold,
+  StartContainerBalancerResponseProto startContainerBalancer(
```

Comment:
I just realised this - changing the method return type here would make this backward incompatible, right?

Reply:
This is not a backward incompatible change as the change is only on the client side. Server was returning StartContainerBalancerResponseProto before as well.

Reply:
Oh yes, makes sense. Thanks for clearing it up!